typescript-selenium/pageobjects/dashboard/Dashboard.ts

```diff
@@ -0,0 +1,64 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { inject, injectable } from "inversify";
+import "reflect-metadata";
+import { TYPES, CLASSES } from "../../inversify.types";
+import { Driver } from "../../driver/Driver";
+import { WebElementCondition, By } from "selenium-webdriver";
+import { DriverHelper } from "../../utils/DriverHelper";
+import { TestConstants } from "../../TestConstants";
+
+@injectable()
+export class Dashboard {
+    private static readonly DASHBOARD_BUTTON_CSS: string = "#dashboard-item";
+    private static readonly WORKSPACES_BUTTON_CSS: string = "#workspaces-item";
+    private static readonly STACKS_BUTTON_CSS: string = "#stacks-item";
+    private static readonly FACTORIES_BUTTON_CSS: string = "#factories-item";
+    private static readonly LOADER_PAGE_CSS: string = ".main-page-loader"
+
+    constructor(@inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper) { }
+
+    async openDashboard(timeout = TestConstants.TS_SELENIUM_LOAD_PAGE_TIMEOUT) {
+        await this.driverHelper.navigateTo(TestConstants.TS_SELENIUM_BASE_URL)
+        await this.waitPage(timeout)
+    }
+
+    async waitPage(timeout = TestConstants.TS_SELENIUM_LOAD_PAGE_TIMEOUT) {
+        await this.driverHelper.waitVisibility(By.css(Dashboard.DASHBOARD_BUTTON_CSS), timeout)
```

Comment:
You are testing that the menu is loaded properly - but it is possible that menu is loaded but the "Dashboard" page is not. I think it would be more safe to add check also for the "Dashboard" title. I do not think the title will change often.